crates/ibc/src/applications/transfer/amount.rs

```diff
@@ -19,6 +19,10 @@ impl Amount {
     pub fn checked_sub(self, rhs: Self) -> Option<Self> {
         self.0.checked_sub(rhs.0).map(Self)
     }
+    
+    pub fn as_u256(&self) -> &U256 {
```

Comment:
We should either make the internal type public, or expose just the methods we want from a `U256`. What methods do you need from the `U256` that are currently not exposed?

Reply:
For me, I want to get the internal type of u256.